tests/Makefile.am

```diff
@@ -154,7 +154,8 @@ command_line_options_DEPENDENCIES = \
 	command-line-options.src/fshort-variable.at \
 	command-line-options.src/java-package.at \
 	command-line-options.src/edit-code-command.at \
-	command-line-options.src/file-path.at
+	command-line-options.src/file-path.at \
+	command-line-options.src/fdefaultbyte.at
```

Comment:
you likely want to add the new textfiles here, as done with `data-rep.src/numeric-dump.cob` above